pkg/mixin/pkgmgmt.go

```diff
@@ -77,3 +78,9 @@ var _ pkgmgmt.PackageMetadata = Metadata{}
 
 // Metadata about an installed mixin.
 type Metadata = pkgmgmt.Metadata
+
+// GetMixinDirectory returns a directory/list of mixins available to install
+func GetMixinDirectory() *packr.Box {
```

Comment:
suggestion: Since we very recently had a function named this, to help people find the directory where a mixin was installed, do you mind renaming this function to `GetMixinDirectoryListings` or something like that. Just to help avoid mixing up the old function and this new one when reading code?

Reply:
Good call, definitely.